blog/_src/posts/2016-11-02-beta-reduction-part-1.md

```diff
@@ -48,7 +53,7 @@ These rules mean that if you have a call in which you have a function applied to
 
 # What is β-reduction?
 
-More generally, what is reduction?  Intuitively, a reduction system is a set of rules that determine how a term is stepped forwards in the computation.  β-reduction is probably the most common reduction system used in programming languages.  It is essentially just the substitution semantics we saw above.  A single β-reduction is as follows
+More generally, what is reduction?  Intuitively, a reduction system is a set of rules that determine how a term is stepped forwards in the computation.  β-reduction is Church's name for removing the λ from a function and substituting the argument for the parameter in the function body.  More formally, we can define β-reduction as follows:
```

Comment:
I find that we should keep using the modern terminology, which is what people will encounter in the wild. Remarks on the historical names could go in a separate paragraph.